### PR TITLE
docs: add tablet section

### PR DESCRIPTION
**Pull request by jp7677**
I forgot those earlier...

Closes https://github.com/labwc/labwc/issues/1401

### CODE REVIEW

File: docs/labwc-config.5.scd [hunk 1]
```diff
@@ -433,6 +433,43 @@ windows using the mouse.
...
+  <map button="stylus" to="right" />
+  <map button="stylus2" to="middle" />
+</tablet>
+```
+
+*<tablet><area top="mm" left="mm" width="mm" height="mm" />*
+	The active tablet area can be specified by setting the *top*/*left*
+	coordinate (in mm) and/or *width*/*height* (in mm). If width or
+	height are omitted or default (0.0), width/height will be set to
+	the remaining width/height seen from top/left.
```
Comment (Consolatis) [inline docs/labwc-config.5.scd:451]: Can we start this with a short explanation on why this setting may be useful for the user and that its completely optional?

File: docs/labwc-config.5.scd [hunk 2]
```diff
@@ -433,6 +433,43 @@ windows using the mouse.
...
+	- stylus2
+	- stylus3
+	- pad
+	- pad2..pad9
+
+	Supported *to* mouse buttons are:
+	- left
+	- right
+	- middle
+
```
Comment (Consolatis) [inline docs/labwc-config.5.scd:472]: We mostly have a "Default is xxx." as the last sentence, for the area this doesn't seem necessary but IMHO for `rotate` and `map` that would be useful.

**Comment by jp7677** [comment]
@Consolatis I've extended the documentation, please let me know if this fits what you had in mind.

File: docs/labwc-config.5.scd [hunk 3]
```diff
@@ -433,6 +433,57 @@ windows using the mouse.
...
+	The *area* element can be used to truncate the active area of the
+	tablet surface. By truncating the active area, it is e.g. possible
+	to maintain the same aspect ratio between output and tablet.
+
+	The active tablet area can be specified by setting the *top*/*left*
+	coordinate (in mm) and/or *width*/*height* (in mm). If width or
+	height are omitted or default (0.0), width/height will be set to
+	the remaining width/height seen from top/left.
+
+	Aspect ratio example:
```
Comment (jp7677) [inline docs/labwc-config.5.scd:458]: I'm fine ofc to remove that part if that is to much for this document.

File: docs/labwc-config.5.scd (hunk 1 above)
Comment (johanmalm) [reply docs/labwc-config.5.scd]: Done

File: docs/labwc-config.5.scd (hunk 2 above)
Comment (johanmalm) [reply docs/labwc-config.5.scd]: Done

**Comment by johanmalm** [comment]
Thanks. Looks good. Comments by @Consolatis actioned.

**Pull request merged**